src/uct/ib/base/ib_device.c

```diff
@@ -196,6 +196,15 @@ static void uct_ib_device_get_locality(const char *dev_name,
     *numa_node = (status == UCS_OK) ? n : -1;
 }
 
+/* dev->async_event_lock must be held */
```

Comment:
libfabric added a debug mode for locks and checks that locks are held correctly with asserts.

https://github.com/ofiwg/libfabric/pull/6677/files#diff-aae9c5738ad1bed4beddaecba3cc3aa6c30ede2e6841ddfcca801b89660ae7bdR220

Reply:
thanks! added